server/router.go

```diff
@@ -29,9 +31,14 @@ func NewPubSubRouter() *PubSubRouter {
 		subscribeChan:   make(chan SubscriptionRequest, 10),
 		unsubscribeChan: make(chan SubscriptionRequest, 10),
 		stop:            make(chan bool, 1),
+		accessManager:   NewAllowAllAccessManager(true),
```

Comment:
I would prefer to
a) leave the accessManager to nil, here and race a panic on Go(), if it is not initialized.
or
b) supply it in the constructor, already